srcpkgs/anki/template

```diff
@@ -1,28 +1,24 @@
 # Template file for 'anki'
 pkgname=anki
-version=2.0.52
+version=2.1.4
 revision=1
 noarch=yes
-depends="python-PyQt4-webkit python-httplib2 python-SQLAlchemy"
+depends="python3-PyQt5-webkit python3-httplib2 python3-SQLAlchemy
```

Comment:
Anki 2.1 uses requests instead of httplib2 and qtwebengine instead of qtwebkit.

Reply:
open a new PR